src/data/facit.csv

```diff
@@ -1,151 +1,151 @@
-N           ,Lagercrantz Group                    ,https://www.lagercrantz.com/sites/lagercrantz/files/files/Lagercrantz%20Group%20H%C3%A5llbarhetsrapport%202022_23_1.pdf                                                                                     ,https://www.lagercrantz.com/sites/lagercrantz/files/files/Lagercrantz%20Group%20H%C3%A5llbarhetsrapport%202022_23_1.pdf                                                                                                                                 ,Brutet räkenskapsår                                                                                                                          ,Nej                                                                                                     ,Nej - rapporten läses fortfarande snett     ,                                                                              
```

Comment:
Random detail (maybe not relevant at all):

The same 2023 report link is used for both 2022 and 2023 for AAK: https://www.aak.com/siteassets/sustainability/sustainability-reports/aak-pai-report-2023.pdf

Does this mean the report was not found for 2022?

Reply:
Good catch, this was handled by @Almendra-ab  now.